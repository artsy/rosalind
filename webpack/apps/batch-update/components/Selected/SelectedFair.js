import React from 'react'
import SelectedComponent from './SelectedComponent'

const SelectedFair = (props) => {
  return (
    <div>
      <h2>Fair</h2>
      <SelectedComponent
        stateKey='fair'
        name={props.name}
        onRemove={props.clearState}
      />
    </div>
  )
}

export { SelectedFair }
