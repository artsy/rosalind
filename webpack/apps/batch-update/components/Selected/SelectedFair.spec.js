import React from 'react'
import renderer from 'react-test-renderer'
import { SelectedFair } from './SelectedFair'

let props

beforeEach(() => {
  props = {
    name: 'Frieze',
    onRemove: jest.fn()
  }
})

it('renders correctly', () => {
  const rendered = renderer.create(<SelectedFair {...props} />)
  const tree = rendered.toJSON()
  expect(tree).toMatchSnapshot()
})
