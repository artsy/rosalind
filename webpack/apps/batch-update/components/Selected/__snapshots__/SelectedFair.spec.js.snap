exports[`test renders correctly 1`] = `
<div>
  <h2>
    Fair
  </h2>
  <div
    className="crrNnH">
    Frieze
    <a
      className="remove"
      href="#"
      onClick={[Function]}>
      âœ•
    </a>
  </div>
</div>
`;
